src/release/src/markdown/RUNNING.txt

```diff
@@ -12,7 +12,7 @@ In order to install and run this application, follow these steps:
 
 1. Download and install a Java Development Kit
```

Comment:
```suggestion
1. Download and install a Java Runtime Environment
```

Reply:
Hum... JRE won't provide the tools to use the heap dump tool in the UI though (it cannot be done programmatically, we have to call jmap on the command line). Nothing wrong with using a JRE otherwise, but people need to be aware that they'll have less diagnostics available from the GUI.

Reply:
We document as only JRE needed; if that is changed by the heap dump tool then it should fail gracefully or something.

Reply:
Yes, you'll get an error message rather than the heap dump summary.